### PR TITLE
add documentation on new HPC-UGent Tier-2 clusters donphan + gallade (replacing slaking + kirlia)

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

File: docs/intro-HPC/2023/donphan-gallade.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+* RHEL8 as operating system;
+
+To start using this cluster from a terminal session, first run:
+```
+module swap cluster/donphan
+```
+
+You can also start (interactive) sessions on `donphan` using the [HPC-UGent web portal](../web_portal.md).
+
+### User limits & overloading on `donphan`
```
Comment (stdweird) [inline docs/intro-HPC/2023/donphan-gallade.md:39]: `&` -> `and`
`overloading` -> `oversubscription`

File: docs/intro-HPC/2023/donphan-gallade.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+By imposing strict user limits and using oversubscription on this cluster,
+we ensure that anyone can quickly get a job with limited resources running, without having to wait in the queue.
```
Comment (stdweird) [inline docs/intro-HPC/2023/donphan-gallade.md:42]: `quickly get a job with limited resources running, without having to wait in the queue` -> 
`get a job running without having to wait in the queue, albeit with limited resources` (yes, albeit ;)

File: docs/intro-HPC/2023/donphan-gallade.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* max. 27GB of memory in total for running jobs;
+
+The job scheduler is configured with to allow *oversubscription* of the available cores,
+which means that jobs will continue to start even if all cores are already occupied by running jobs.
+While this prevents waiting time in the queue, it does imply that performance will degrade when all cores are occupied
+and additional jobs continue to start running.
+
+### Shared GPU on `donphan` workernodes
+
+Each `donphan` workernode includes a single NVIDIA A2 GPU that can be used for compute workloads,
```
Comment (stdweird) [inline docs/intro-HPC/2023/donphan-gallade.md:57]: `compute` -> `light compute`

File: docs/intro-HPC/2023/donphan-gallade.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+and to accelerate certain graphical tasks.
+
+This GPU is *shared* across all jobs running on the workernode, and does not need to be requested explicitely
```
Comment (stdweird) [inline docs/intro-HPC/2023/donphan-gallade.md:60]: `explicitely` -> `explicitly`

File: docs/intro-HPC/2023/donphan-gallade.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+* 1.5TB NVME local disk;
+* HDR-100 InfiniBand interconnect;
+* RHEL8 as operating system;
+
+To start using this cluster from a terminal session, first run:
+```
+module swap cluster/gallade
+```
+
+You can also start (interactive) sessions on `gallade` using the [HPC-UGent web portal](../web_portal.md).
```
Comment (stdweird) [inline docs/intro-HPC/2023/donphan-gallade.md:92]: This section needs a firm warning that compiled code on kirlia (eg conda) will fail with horrendous looking error messages.

also point out that besides very different cpu arch, also the memory per core is much lower; and total number of cores is much higher (eg when using `ppn=all`)

**Comment by boegel** [comment]
@stdweird All suggestions tackled in 04576bf

**Pull request merged**